packages/loot-core/src/platform/server/fetch/index.electron.ts

```diff
@@ -1,12 +1,20 @@
 // @ts-strict-ignore
 import nodeFetch from 'node-fetch';
 
-export const fetch = (input: RequestInfo | URL, options?: RequestInit) => {
-  return nodeFetch(input, {
-    ...options,
-    headers: {
-      ...options?.headers,
-      origin: 'app://actual',
-    },
-  });
+export const fetch = async (
+  input: RequestInfo | URL,
+  options?: RequestInit,
+) => {
+  try {
+    return await nodeFetch(input, {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏the `async`/`await` here is unnecessary
```suggestion
export const fetch = (
  input: RequestInfo | URL,
  options?: RequestInit,
) => {
  try {
    return nodeFetch(input, {
```

Reply:
When I do this the logs don't show because the promise will always return successfully (because it hasn't resolved/errored yet)

![no log](https://github.com/user-attachments/assets/d2f15052-91f7-40d8-bffe-6626641b3f1f)